master/docs/manual/configuration/buildsteps.rst

```diff
@@ -385,6 +385,7 @@ The Git step takes the following arguments:
 ``progress`` (optional)
    Passes the (``--progress``) flag to (:command:`git fetch`).
    This solves issues of long fetches being killed due to lack of output, but requires Git 1.7.2 or later.
+   Its value is True on Git 1.7.2 or later.
```

Comment:
please don't default this value to true on recent git. It defaults to false not for compat reason, but for performance reason. 
If we set progress to true, then suddently a lot more unneeded log is done.

Reply:
I wonder whether the extra logs are actually a performance problem. On long running fetches the 1-per-second ping will be the most significant factor which results in roughly 3600 lines of log per hour or something like 300kB. Compiling small C++ projects with compiler warnings produces more log output than that. Cloning small project like Buildbot produces only several hundreds lines of log.

By the way, other CIs like Travis or Circle enables this behavior by default by presenting a fake a terminal for stderr.